[2025-09-01T05:51:42.243+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-09-01T05:51:33.590672+00:00 [queued]>
[2025-09-01T05:51:42.251+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-09-01T05:51:33.590672+00:00 [queued]>
[2025-09-01T05:51:42.251+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-01T05:51:42.266+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): archive> on 2025-09-01 05:51:33.590672+00:00
[2025-09-01T05:51:42.273+0000] {standard_task_runner.py:60} INFO - Started process 569 to run task
[2025-09-01T05:51:42.277+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'archive', 'manual__2025-09-01T05:51:33.590672+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp87ym9ncz']
[2025-09-01T05:51:42.279+0000] {standard_task_runner.py:88} INFO - Job 23: Subtask archive
[2025-09-01T05:51:42.311+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-01T05:51:42.414+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.archive manual__2025-09-01T05:51:33.590672+00:00 [running]> on host 857434650c3d
[2025-09-01T05:51:42.574+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='archive' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T05:51:33.590672+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-01T05:51:33.590672+00:00'
[2025-09-01T05:51:42.576+0000] {logging_mixin.py:188} INFO - Running task: ARCHIVE
[2025-09-01T05:51:42.576+0000] {logging_mixin.py:188} INFO - Simulating the archival of the original raw data...
[2025-09-01T05:51:42.576+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-01T05:51:42.590+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=archive, execution_date=20250901T055133, start_date=20250901T055142, end_date=20250901T055142
[2025-09-01T05:51:42.631+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-01T05:51:42.655+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
